ocfweb/lab_reservations/templates/lab_reservations/pending.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <div class="ocf-content-block">
+        <p>Your lab reservation request has successfully
+           been submitted and is pending approval!</p>
```

Comment:
nit: i'd encourage writing multi-line elements like:

```html
<p>
    Your lab reservation request has been successfully
    submitted and is pending approval!
</p>
```

for the same reason behind most of our python style guidelines: it reduces diff churn, which makes git blame, merge conflicts, etc. better